src/pudl/output/ferc1.py

```diff
@@ -1782,13 +1786,17 @@ def prep_table_to_explode(
     def boom(self: Self, tables_to_explode: dict[str, pd.DataFrame]) -> pd.DataFrame:
         """Explode a set of nested tables.
 
-        There are five main stages of this process:
+        There are seven main stages of this process:
 
-        #. Prep all of the individual tables for explosion.
-        #. Concatenate all of the tabels together.
-        #. Remove duplication in the concatenated exploded table.
-        #. Annotate the fine-grained data with additional metadata.
-        #. Validate that calculated top-level values are correct. (not implemented)
+        #. Prep all of the individual tables for explosion (via :meth:`prep_table_to_explode`).
+        #. Concatenate all of the tabels together (via :meth:`initial_explosion_concatenation`).
+        #. Add special case calculation corrections when ``xbrl_factoid`` are off by one other
+           ``xbrl_factoid`` (via :meth:`add_sizable_minority_corrections`).
+        #. Reconcile the inter-table calculations (via :meth:`reconcile_intertable_calculations`)
+        #. Annotate the data with additional metadata (via:meth:`XbrlCalculationForestFerc1.annotated_forest`).
+        #. Identify the most fine-grained ``xbrl_factoids`` (via
+           :meth:`XbrlCalculationForestFerc1.leafy_meta`).
+        #. Validate that calculated top-level values are correct (not implemented).
```

Comment:
i updated these steps with links to all of the main stages of the explosion process so if someone was curious about the guts they could attempt to follow along. documenting the full gory detail of all of these step in one place seems generally like it would be very overwhelming, so I kept it list-y and reference-y